recipes/daggy/2.1/conanfile.py

```diff
@@ -28,7 +28,8 @@ class DaggyConan(ConanFile):
         "shared": False,
         "fPIC": True
     }
-    generators = "cmake", "cmake_find_package"
+    generators = "cmake", "cmake_paths", "cmake_find_package"
+    tool_requires = "cmake/3.23.1"
```

Comment:
```suggestion
```
Revert. Use build_requirements. You can update to `self.tool_requires("cmake/3.23.1")`